wordpress-mobile/WordPress-iOS
# Invite Links: Changes blog.inviteLinks relation deletion rule to cascade

Refs #15356 

I noticed that the deletion rule for the inviteLinks relation on the blog model was inadvertently set to Deny rather than Cascade.  I probably miss clicked this in #15940.   This PR corrects the deletion rule.

Ideally we should merge this before #15970. 

To test:
Confirm the app builds.  
Visually confirm the correct deletion rule is selected.

PR submission checklist:
